odoo_backup_sh/doc/index.rst

```diff
@@ -25,19 +25,40 @@ Configuration
   server_wide_modules = web,odoo_backup_sh
   # (...)
 
+In order for desired databases to appear in a backup dashboard, you have to create settings for it.
+Open the menu ``[[ {Backups} ]] >> {Databases} >> {Databases}``. Click on the ``[{Create}]`` button.
+Fill required fields:
 
-.. todo:: Add the section when interfaces are ready.
-{Instruction how to configure the module before start to use it}
+- Database: select one of your databases
+- Auto Backup Execute Every:  type a value and select a unit of measure.
+  Example: You want the module to make new backup every 2 hours.
+  If you don't want use auto backup mode then leave this field blank.
+- Auto Rotation: If you have set up the auto backup, you can configure the scheme of storing your backups on
+  a remote storage. You can specify how many backups to preserve for certain time frames.
+  Example: The module makes auto backup your database every 2 hours. You want to preserve 2 daily backups and 1 weekly.
+  Just put this numbers in appropriate fields. Then you will have one last backup for each week and two last backups for
+  each day.
```

Comment:
It doesn't work this way.  `` 2 daily backups`` means to have one backup per each day for last two days, not **two last backups for each day.**
Correct:
* 1 backup from day1
* 1 backup from day2

Incorrect:
* 2 backups from day1
* 2 backup2 from day2